hypothesis-python/tests/cover/test_sampled_from.py

```diff
@@ -48,9 +51,20 @@ def test_can_sample_ordereddict_without_warning():
     sampled_from(an_ordereddict).example()
 
 
-@given(sampled_from(an_enum))
-def test_can_sample_enums(member):
-    assert isinstance(member, an_enum)
+@pytest.mark.parametrize("enum_class", [an_enum, a_flag, an_empty_flag])
+def test_can_sample_enums(enum_class):
+    @given(sampled_from(enum_class))
+    def test_it(member):
+        assert isinstance(member, enum_class)
+
+    test_it()
+
+
+@pytest.mark.skipif(sys.version_info[:2] < (3, 12), reason="behaviour change")
+def test_invalid_enum():
+    not_constructible_enum = enum.Flag("invalid", ())
+    with pytest.raises(InvalidArgument):
+        st.sampled_from(not_constructible_enum).example()
```

Comment:
After the latest commit, I don't think this test is necessary for coverage anymore - it just defers to the generic sample-empty-sequence error. Since the future cpy behaviour is not certain, should I just drop this test?

Reply:
I dropped it